ui/src/views/Documentation/Reference/ApiReference.jsx

```diff
@@ -18,6 +41,23 @@ export default class ApiReference extends Component {
     apiVersion: string.isRequired,
   };
 
+  groupBy = (list, keyGetter) => {
+    const map = new Map();
```

Comment:
Nice usage of Map :)

Reply:
In particular, I'm pleased that Map retains the order in which its keys were inserted, so that we can adjust the order of categories in the display by adjusting the order in the reference files.

Reply:
Thanks for the review! I had fun doing this part because it felt like solving an algorithm problem. It helped me realize that algorithm problems can be applied to real-world solutions and that was so exciting!